### DISCUSSION TITLE
The repository [name] has been retired and cannot be reused. What can I do?

### DISCUSSION THREAD

**Discussion by dvakatriii** [community/community#23480] [Other Feature Feedback, Questions, & Ideas]
<p>I want to go back to my <strong>old</strong> nickname and <strong>create a README for that nickname</strong>, but I get this error. What can I do?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>you are changing your username very fast, it will take time it’s like an identity that you should not be changing from time to time</p>
<p>try contacting them but there is no assurance or may take longer</p>
<p><a href="https://support.github.com/contact/account">https://support.github.com/contact/account</a></p>

**Comment by longmen06** [comment]
why do i get error Repository name [repo/directory] has been retired and cannot be reused
